Makefile

```diff
@@ -14,21 +19,27 @@ integration: test
 	@go test -tags=integration
 
 linux:
-	env GOOS=linux GOARCH=amd64 go build -o benchmarks-linux-amd64
+	@env GOOS=linux GOARCH=amd64 go build -o benchmarks-linux-amd64
 
 mac:
-	env GOOS=darwin GOARCH=amd64 go build -o benchmarks-mac-amd64
+	@env GOOS=darwin GOARCH=amd64 go build -o benchmarks-mac-amd64
 
 windows:
-	env GOOS=windows GOARCH=amd64 go build -o benchmarks-windows-amd64.exe
+	@env GOOS=windows GOARCH=amd64 go build -o benchmarks-windows-amd64.exe
 
 $(glidepath)/glide:
-	git clone https://github.com/Masterminds/glide.git $(glidepath)
-	cd $(glidepath);make build
-	cp $(glidepath)/glide .
+	@git clone https://github.com/Masterminds/glide.git $(glidepath)
+	@cd $(glidepath);make build
+	@cp $(glidepath)/glide .
+
+$(GOPATH)/bin/easyjson:
```

Comment:
Shouldn't glide fetch this dependency instead?

Reply:
Indeed, we can try

Reply:
Add this change and I you might need to rebase but then I think we can merge this PR :)